rotkehlchen/tests/api/test_nfts.py

```diff
@@ -135,3 +135,20 @@ def test_nft_ids_are_unique(rotkehlchen_api_server):
     all_ids = ids_1 + ids_2
     set_of_ids = set(all_ids)
     assert len(all_ids) == len(set_of_ids)
+
+
+@flaky(max_runs=3, min_passes=1)  # all opensea calls have become quite flaky
+@pytest.mark.parametrize('ethereum_accounts', [[TEST_ACC2, TEST_ACC3]])
+@pytest.mark.parametrize('start_with_valid_premium', [True])
+@pytest.mark.parametrize('ethereum_modules', [['nfts']])
+def test_nft_balances_include_collection_name(rotkehlchen_api_server):
+    """Check that nfts balances always include a `collection_name`"""
+    response = requests.get(api_url_for(
+        rotkehlchen_api_server,
+        'nftsbalanceresource',
+    ), json={'async_query': False})
+
+    result = assert_proper_response_with_result(response)
+    for nfts_balances in result.values():
+        for nft_balance in nfts_balances:
+            assert 'collection_name' in nft_balance
```

Comment:
maybe would be better to query an account like mine `yabir.eth` and check that all the fields are there and are the expected ones (you can search for the ens nft and verify that the output is the expected)